### PR TITLE
Visualization bugfix

### PR DESCRIPTION
**Pull request by szok**
Added:
- Bugfix moving Rack in visualization
- Breadcrumbs to visualization
- Barcode/sn validation
- Visualization links

### CODE REVIEW

File: src/ralph/assets/models/assets.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from dateutil.relativedelta import relativedelta
 from mptt.models import MPTTModel, TreeForeignKey
 
+from django.core.exceptions import ValidationError
```
Comment (ar4s) [inline src/ralph/assets/models/assets.py]: sort and move `ValidationError` two lines above

File: src/ralph/dc_view/serializers/models_serializer.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 import six
 
+from collections import OrderedDict
```
Comment (mkurek) [inline src/ralph/dc_view/serializers/models_serializer.py:8]: collections is builtin module (as opposed to six) - fix imports order

File: src/ralph/dc_view/serializers/models_serializer.py [hunk 3]
```diff
@@ -121,13 +125,14 @@ class Meta:
...
     def get_type(self, obj):
         return TYPE_PDU
 
     class Meta:
         model = DataCenterAsset
-        fields = ('model', 'sn', 'orientation')
+        fields = ('model', 'sn', 'orientation', 'url')
 
 
 class RackSerializer(AdminLinkMixin, serializers.ModelSerializer):
```
Comment (ar4s) [inline src/ralph/dc_view/serializers/models_serializer.py]: please separate this serializer into: `RackBaseSerializer` and `RackCreateSerializer` (which inherit from `RackBaseSerializer`)

**Comment by ar4s** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**